NEWS.md

```diff
@@ -1,6 +1,9 @@
 
 # vctrs 0.2.0.9000
 
+* `vec_cast()` now casts data frames to lists row wise to preserve the invariant
```

Comment:
Suggestion:

`vec_cast()` now casts data frames to lists rowwise, i.e. to a list of data frames of size 1. This preserves the invariant of ...